packages/ember-glimmer/lib/utils/string.js

```diff
@@ -0,0 +1 @@
+../../../ember-htmlbars/lib/utils/string.js
```

Comment:
I think this should go the other way since in the long term we are going to `rm -rf packages/ember-htmlbars`.